docs/docs/0.3.0/policies/README.md

```diff
@@ -209,13 +208,143 @@ spec:
         service: redis
         version: '2.0'
 ```
-You also need to add labels to your services and use that label as a matching tag. Above, you  see that we utilize the `version` tag to help with canary deployment. Another common use-case is to add a `env` tag as you separate testing, staging, and production environments' services.
 
-#### Resolving Collision in Traffic Routes
-In the event where two traffic route many have colliding configurations, these are the expected behaviors:
-* Identical Selectors: the policy will be prioritized alphabetically by name
-* More tags: The policy with more matching tags will be prioritized first
-* Specific tags: The policy that match by exact value will be prioritized over ones matched by `'*'` 
+The expected outcome is that 1 out of roughly 10 request will be routed to the new version 2. 
+
+### Resolving Collision in Traffic Routes
```

Comment:
Let's move section `Resolving Collision in Traffic Routes` into a separate PR.

The idea is that in `0.3.1` release we will now have 3 resources with identical behaviour of conflict resolution:
* `TrafficRoute`
* `TrafficLog`
* `HealthCheck`

So, it makes sense to change the way we describe it. It's no longer `TrafficRoute`-specific.